app/src/main/AndroidManifest.xml

```diff
@@ -68,7 +68,7 @@
             android:windowSoftInputMode="adjustPan" />
 
         <activity
-            android:name=".view.createWallet.CreateWalletActivity"
+            android:name=".view.bootWallet.BootWalletActivity"
```

Comment:
naming convention for package names is all lower case.
i.e. bootwallet

Reply:
👍